src/Uno/Prism.Uno/Prism.Uno.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;xamarinios10;monoandroid90;xamarinmac20;uap10.0.16299</TargetFrameworks>
```

Comment:
Thoughts on splitting out the UWP target on Mac/Windows?

https://github.com/xamarin/Essentials/blob/b7da1f50e227cf12637b8018bc1025a9cfdee677/Xamarin.Essentials/Xamarin.Essentials.csproj#L3-L4

Reply:
The splitting here is only useful when building the library on Linux or macOS, and only the library. The library is generally built transitively when building an app, and the other targets are not build on macOS. The final nuget package needs to be built on Windows for that reason.

But it does not hurt to make the change, I'll add it.